Apromore-Core-Components/Apromore-BPMNEditor/src/main/scripts/editor.js

```diff
@@ -56,8 +56,32 @@ Apromore.Editor = {
         return Ext.get(this.rootNode).parent("div{overflow=auto}", true);
     },
 
+    updateUndoRedo: function () {
+        try {
+            var undo = jq("button[title='Undo the last action']");
+            var redo = jq("button[title='Redo the last undone action']");
```

Comment:
This free way of element selection can be improved via a clean API in the future.